runtime.js

```diff
@@ -460,10 +460,12 @@
 
       // Pre-initialize at least 20 temporary variables to enable hidden
       // class optimizations for simple generators.
```

Comment:
Performance on that is quite bad when used without the flag, causing the performance relative to my "ES5 baseline" to drop from 1/4 to 1/26th. Perhaps that combined with the flag is the best option?